ogticrd/covid-safe-paths
# refactoring phone regex

#### Description:


Fixed the problem with deleting phone input, when you tried to delete a phone number the regex did not let you delete it one by one

#### Linked issues:


Fix: [149](https://trello.com/c/oH8j3Q2i/149-no-se-puede-borrar-completamente-el-numero-de-tel%C3%A9fono-en-el-form)
#### Screenshots:



#### How to test:


- `git pull && git checkout Fix/149-deleting-phoneNumber-correctly`
- Head over to report screen and press `Start with Dominican ID`
- Enter a phone number and try to delete it one by one, it should let you do it
